org.grisbi.Grisbi.yml

```diff
@@ -0,0 +1,60 @@
+app-id: org.grisbi.Grisbi
+runtime: org.gnome.Platform
+runtime-version: "44"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.gnome.Sdk.Locale
+command: grisbi
+
+rename-desktop-file: grisbi.desktop
+rename-icon: grisbi
+
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --filesystem=xdg-download
+
+cleanup:
+  - '/include'
+  - '*.a'
+
+modules:
+
+  - shared-modules/intltool/intltool-0.51.json
+
+  - name: libgsf
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/libgsf/1.14/libgsf-1.14.50.tar.xz
+        sha256: 6e6c20d0778339069d583c0d63759d297e817ea10d0d897ebbe965f16e2e8e52
+
+  - name: opensp
+    config-opts:
+      - --disable-doc-build
+    sources:
+      - type: archive
+        url: http://downloads.sourceforge.net/project/openjade/opensp/1.5.2/OpenSP-1.5.2.tar.gz
+        sha256: 57f4898498a368918b0d49c826aa434bb5b703d2c3b169beb348016ab25617ce
```

Comment:
this needs also a "shell" source with the following command: `"autoreconf -vfi"` to regenerate the configure files to be able to build on aarch64